keycloak/sms-provider/src/test/java/org/sunbird/keycloak/rest/RequiredActionLinkProviderTest.java

```diff
@@ -84,15 +84,13 @@ public void checkRealmAdminAccessForUnAuthorized() throws Exception {
       assertTrue(response == null);
     } catch (WebApplicationException ex) {
       assertEquals(ex.getResponse().getStatus(), expectedException.getResponse().getStatus());
-      assertEquals(((ErrorRepresentation) (ex.getResponse().getEntity())).getErrorMessage(),
-          ((ErrorRepresentation) (expectedException.getResponse().getEntity())).getErrorMessage());
     }
   }
 
   @Test
   public void checkRealmAdminAccessForForbidden() throws Exception {
     PowerMockito.whenNew(AppAuthManager.class).withAnyArguments().thenReturn(authMangr);
-    PowerMockito.when(authMangr.authenticateBearerToken(session, model)).thenReturn(authResult);
+    PowerMockito.when(authMangr.authenticateIdentityCookie(session, model)).thenReturn(authResult);
```

Comment:
same comment as before

Reply:
Now this is reverted back with proper method changes, done in the latest commit.